File: E_Kotlin/Projects/AndroidStudio/XML/PoliceDriverGame/app/src/main/java/com/sebss/juegosnake/androidimpl/TouchHandler.java
package com.sebss.juegosnake.androidimpl;

import java.util.List;

import android.view.View.OnTouchListener;

import com.sebss.juegosnake.Input.TouchEvent;


public interface TouchHandler extends OnTouchListener {
    boolean isTouchDown(int pointer);

    int getTouchX(int pointer);

    int getTouchY(int pointer);

    List<TouchEvent> getTouchEvents();
}

File: E_Kotlin/Projects/3DGraphicsMotor/app/src/main/java/com/sebss/a3dgraphicsmotor/androidimpl/TouchHandler.kt
package com.sebss.a3dgraphicsmotor.androidimpl

import android.view.View.OnTouchListener
import com.sebss.a3dgraphicsmotor.interfaces.Input.TouchEvent

interface TouchHandler : OnTouchListener {
    fun isTouchDown(pointer: Int): Boolean
    fun getTouchX(pointer: Int): Int
    fun getTouchY(pointer: Int): Int
    fun getTouchEvents(): List<TouchEvent>
}